I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 17, in script call
    call EventCheck
  File "game/script.rpy", line 28, in script
    if (Hours == 12) and (Minutes == 0) and (Days == 2) and (Month == 0):
  File "game/script.rpy", line 28, in <module>
    if (Hours == 12) and (Minutes == 0) and (Days == 2) and (Month == 0):
NameError: name 'Hours' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 17, in script call
    call EventCheck
  File "game/script.rpy", line 28, in script
    if (Hours == 12) and (Minutes == 0) and (Days == 2) and (Month == 0):
  File "/Users/daveshields/Documents/RenPy/renpy-7.4.9-sdk/renpy/ast.py", line 1898, in execute
    if renpy.python.py_eval(condition):
  File "/Users/daveshields/Documents/RenPy/renpy-7.4.9-sdk/renpy/python.py", line 2276, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "/Users/daveshields/Documents/RenPy/renpy-7.4.9-sdk/renpy/python.py", line 2269, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 28, in <module>
    if (Hours == 12) and (Minutes == 0) and (Days == 2) and (Month == 0):
NameError: name 'Hours' is not defined

Darwin-20.6.0-x86_64-i386-64bit
Ren'Py 7.4.9.2142
Thundy Calendar 1.0
Mon Oct 18 16:50:56 2021
